internal/pkg/describe/static_site.go

```diff
@@ -172,6 +207,15 @@ func (w *staticSiteDesc) HumanString() string {
 			fmt.Fprintf(writer, "  %s\t%s\n", route.Environment, route.URL)
 		}
 	}
+	if len(w.Objects) != 0 {
+		fmt.Fprint(writer, color.Bold.Sprint("\nS3 Bucket Objects\n"))
+		writer.Flush()
+		for _, object := range w.Objects {
+			fmt.Fprintf(writer, "\n  %s\t%s\n", "Environment", object.Environment)
+			fmt.Fprintf(writer, object.Tree)
+			writer.Flush()
```

Comment:
It looks like there is no tab to align, so probably we can flush once after the `for _, object := range w.Objects` loop?